dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -0,0 +1,146 @@
+import { brandBackground, brandLine } from '@guardian/source-foundations';
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+
+import { Lines } from '@guardian/source-react-components-development-kitchen';
+import { SubNav } from '../components/SubNav.importable';
+import { ElementContainer } from '../components/ElementContainer';
+import { Nav } from '../components/Nav/Nav';
+
+import { Island } from '../components/Island';
+import { MostViewedFooterLayout } from '../components/MostViewedFooterLayout';
+import { decidePalette } from '../lib/decidePalette';
+import { ContainerLayout } from '../components/ContainerLayout';
+import { Header } from '../components/Header';
+
+interface Props {
+	Front: FrontType;
+	NAV: NavType;
+}
+
+export const FrontLayout = ({ Front, NAV }: Props) => {
+	const {
+		config: { isPaidContent },
+	} = Front;
+
+	const format = {
+		display: ArticleDisplay.Standard,
+		design: ArticleDesign.Standard,
+		theme: ArticlePillar.News,
+	};
+
+	const palette = decidePalette(format);
+
+	// const contributionsServiceUrl = getContributionsServiceUrl(Front);
+
+	return (
+		<>
+			<div data-print-layout="hide" id="bannerandheader">
+				<>
+					<ElementContainer
+						showTopBorder={false}
+						showSideBorders={false}
+						padded={false}
+						backgroundColour={brandBackground.primary}
+						element="header"
+					>
+						<Header
+							edition={Front.editionId as Edition}
+							idUrl={Front.config.idUrl}
+							mmaUrl={Front.config.mmaUrl}
+							supporterCTA={
+								Front.nav.readerRevenueLinks.header.supporter
+							}
+							discussionApiUrl={Front.config.discussionApiUrl}
+							isAnniversary={
+								Front.config.switches.anniversaryHeaderSvg
+							}
+							urls={Front.nav.readerRevenueLinks.header}
+							remoteHeader={Front.config.switches.remoteHeader}
+							contributionsServiceUrl="https://contributions.guardianapis.com" // TODO: Pass this in
+						/>
+					</ElementContainer>
+					<ElementContainer
+						showSideBorders={true}
+						borderColour={brandLine.primary}
+						showTopBorder={false}
+						padded={false}
+						backgroundColour={brandBackground.primary}
+						element="nav"
+					>
+						<Nav
+							nav={NAV}
+							format={format}
+							subscribeUrl={
+								Front.nav.readerRevenueLinks.header.subscribe
+							}
+							edition={Front.editionId as Edition}
```

Comment:
Could we fix it upstream in the Fronts type?

Reply:
Done